README.md

```diff
@@ -18,6 +18,29 @@ If you are using TypeScript you can install the accompanying types
 npm install --save-dev @types/slug
 ```
 
+## Differences between `slug` and `slugify` packages
+
+I also contribute to a similar package called [`slugify`](https://github.com/simov/slugify).
+Here are the key differences.
+
+- **Defaults:** `slug` has the `lower` option enabled by default, lowercasing all slugs
+  (`'On SALE'` becomes `'on-sale'`).  
+  `slugify` has the `lower` option disabled by default (`'On SALE'` becomes `'On-SALE'`).
+- **IE11 Support:** `slug` currently still supports IE11
+  (_However, [support will be dropped June 2022](https://github.com/Trott/slug/blob/beta/CHANGELOG.md)!_).  
+  `slugify` dropped support for IE11 some time ago.
+- **Currency:** `slug` removes currency symbols (`'$100'` becomes `'100'`).  
+  `slugify` maps them to words (`$100` becomes `dollar100`).
+- **Empty Output:** If an input string contains only spaces, or other unrecognized/unmapped characters,
+  it would result in an empty slug.  
```

Comment:
I think this sentence needs to specify that it's talking about `slugify` behavior.

Reply:
Yeah, I was describing the overall situation, but I can remove it.

Reply:
Updated.